mods/Windows/OnlineCTR/Network_PC/Client/CL_main.c

```diff
@@ -153,6 +153,11 @@ void ProcessReceiveEvent(ENetPacket* packet)
 			break;
 		}
 
+		case SG_BAD_USERNAME:
```

Comment:
printing a message here would make it more useful, as it is now it isn't serving any purpose